.travis.yml

```diff
@@ -3,17 +3,26 @@ language: node_js
 os: linux
 
 matrix:
-  include:   
-    - node_js: '8'
-    - node_js: '10'
+  include:
+  - node_js: '8'
+  - node_js: '10'
 
 sudo: false
 
 install:
-  - npm install
-
+- npm install
 script:
-  - npm test
+- npm test
```

Comment:
should we also add linting verification? Or is that a `pre` call on `test`

Reply:
Exactly. Now that you mention it though, we should be calling `npm run build` so that we run `tsc` as well. Currently, `test` runs as a precursor to `build`. I think I saw that change in your incoming PR, so maybe we can update whichever one comes in last